pkg/image/builder.go

```diff
@@ -515,7 +515,7 @@ func IsPrivateImage(image string, dockerHubRegistry registry.RegistryOptions) (b
 			sysCtx.DockerInsecureSkipTLSVerify = types.OptionalBoolTrue
 		}
 
-		remoteImage, err := ref.NewImage(context.Background(), &sysCtx)
+		remoteImage, err := ref.NewImageSource(context.Background(), &sysCtx)
```

Comment:
NewImage must check local docker or use docker user creds or something cause it wasnt finding private images on my codeserver.

Reply:
It does! Good find.